### ISSUE TITLE
Installation Docs

### ISSUE THREAD

**Issue by ashwini0529** [fossasia/susper.com#30]
Need to make Installation Documentation.

**Comment by Elgayar** [comment]
Can you elaborate a bit more please?

**Comment by ashwini0529** [comment]
https://github.com/fossasia/susper.com/tree/gh-pages/docs

Everything here's empty.

**Comment by Elgayar** [comment]
Oh, I get it, anyways I'd love to work on them but the search engine at susper.com isn't working, and I need it to work so that when I deploy, I can make sure my steps are correct. @ashwini0529

**Comment by ashwini0529** [comment]
Yeah. If you want to take this issue up, you can start documenting right after it starts working again

**Comment by Elgayar** [comment]
I will the second it's up!

**Comment by ashwini0529** [comment]
Alright 👍
